Discord/src/commands/player/ClassesInfoCommand.ts

```diff
@@ -0,0 +1,216 @@
+import {DraftbotInteraction} from "../../messages/DraftbotInteraction";
+import {KeycloakUser} from "../../../../Lib/src/keycloak/KeycloakUser";
+import {
+	CommandClassesInfoPacketReq,
+	CommandClassesInfoPacketRes
+} from "../../../../Lib/src/packets/commands/CommandClassesInfoPacket";
+import {makePacket, PacketContext} from "../../../../Lib/src/packets/DraftBotPacket";
+import {DiscordCache} from "../../bot/DiscordCache";
+import {DraftBotEmbed} from "../../messages/DraftBotEmbed";
+import i18n from "../../translations/i18n";
+import {ICommand} from "../ICommand";
+import {SlashCommandBuilderGenerator} from "../SlashCommandBuilderGenerator";
+import {Effect} from "../../../../Lib/src/enums/Effect";
+import { Constants } from "../../../../Lib/src/constants/Constants";
+import {ClassInfoConstants} from "../../../../Lib/src/constants/ClassInfoConstants";
+import {Language} from "../../../../Lib/src/Language";
+import {
+	ActionRowBuilder,
+	StringSelectMenuBuilder,
+	StringSelectMenuInteraction,
+	StringSelectMenuOptionBuilder
+} from "discord.js";
+import {sendInteractionNotForYou} from "../../utils/ErrorUtils";
+
+function getPacket(interaction: DraftbotInteraction, user: KeycloakUser): Promise<CommandClassesInfoPacketReq> {
+	return Promise.resolve(makePacket(CommandClassesInfoPacketReq, {keycloakId: user.id, language: interaction.userLanguage}));
+}
+
+function getListEmbed(packet: CommandClassesInfoPacketRes, language: Language, classList: {
+	id: number,
+	emoji: string,
+	lng: Language,
+	health: number,
+	attack: number,
+	defense: number,
+	speed: number,
+	baseBreath: number,
+	maxBreath: number,
+	breathRegen: number,
+	fightPoint: number,
+	description: string,
+	attacks: {
+		id: string,
+		cost: number,
+		emoji: string
+	}[]
+}[]): DraftBotEmbed {
+	const embed = new DraftBotEmbed().setTitle(i18n.t("commands:classesInfo.title.list", {
+		lng: language
+	}));
+
+	const classesList = [];
+	for (const classStats of classList) {
+		classesList.push(ClassInfoConstants.FIELDS_VALUE
+			.replace("{name}", `${classStats.emoji} ${i18n.t(`models:classes.${classStats.id}`, {
+				lng: classStats.lng
+			})}`)
+			.replace("{health}", classStats.health.toString())
+			.replace("{attack}", classStats.attack.toString())
+			.replace("{defense}", classStats.defense.toString())
+			.replace("{speed}", classStats.speed.toString())
+			.replace("{baseBreath}", classStats.baseBreath.toString())
+			.replace("{maxBreath}", classStats.maxBreath.toString())
+			.replace("{breathRegen}", classStats.breathRegen.toString())
+			.replace("{fightPoint}", classStats.fightPoint.toString()));
+	}
+
+	embed.setDescription(`${i18n.t("commands:classesInfo.description.list", {
+		lng: language
+	})}\n\n${classesList.join("\n")}`);
+
+	return embed;
+}
+
+// Todo: edit the function's instructions to match the actual implementation
```

Comment:
Je ne comprends pas trop ce que veut dire "the function's instructions". Et quelle implémentation ?

Reply:
C'était un todo que j'ai oublié de retirer.